docs/contributors/document.md

```diff
@@ -63,6 +63,29 @@ To create links that work in all contexts, you should use absolute path links wi
 
 This way they will be properly handled in all three aforementioned contexts.
 
+### Code Examples
+
+The code example in markdown should be wrapped in three tick marks <code>```</code> and can additionally include a language specifier. See this [Github documentation around fenced code blocks](https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks).
```

Comment:
```suggestion
The code example in markdown should be wrapped in three tick marks <code>```</code> and can additionally include a language specifier. See this [GitHub documentation around fenced code blocks](https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks).
```